.github/workflows/visual-regression-tests.yml

```diff
@@ -20,3 +20,24 @@ jobs:
         run: npm run visreg-ci
         env:
           PERCY_TOKEN: ${{ secrets.PERCY_TOKEN }}
+  cli-verification:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Setup Node
+        uses: actions/setup-node@v2
+        with:
+          node-version: '16.10'
```

Comment:
Quick question on this. Emulsify CLI technically requires 16.13 or greater right? Does this work here?

Reply:
It does. I think it just needs to be v16... I could update it to require the same thing, but I think we're going to be chasing our tails if we try to keep everything (Sous, emulsify-drupal, Compound, CLI, etc.) in 100% step at all times.

If we want to do that, we'll need to set up some kind of process to communicate node version changes across the projects